cmd/capdctl/main.go

```diff
@@ -47,6 +47,21 @@ func (mo *machineOptions) initFlags(fs *flag.FlagSet) {
 	mo.version = fs.String("version", "v1.14.2", "The Kubernetes version to run")
 }
 
+type machineDeyploymentOptions struct {
+	name, namespace, clusterName, controlPlaneVersion, kubeletVersion *string
+	replicas                                                          *int32
+}
+
+func (mo *machineDeyploymentOptions) initFlags(fs *flag.FlagSet) {
+	mo.name = fs.String("name", "my-machine", "The name of the machine")
+	mo.namespace = fs.String("namespace", "my-namespace", "The namespece of the machine")
+	mo.clusterName = fs.String("cluster-name", "my-cluster", "The name of the cluster the machine belongs to")
+	mo.controlPlaneVersion = fs.String("control-plane-version", "v1.14.2", "The Kubernetes control plane version to run")
+	mo.kubeletVersion = fs.String("kubelet-version", "v1.14.2", "The Kubernetes kubelet version to run")
+	replicas := int32(*fs.Int("replicas", 1, "The number of replicas"))
```

Comment:
god i love go so much.

I would probably recommend passing in an int here for readability and converting it inside the machine deployment function.

but it doesn't really matter, that's just a minor style preference